### ISSUE TITLE
Build error in bigartm CLI - lunwind library

### ISSUE THREAD

**Issue by sashafrey** [bigartm/bigartm#354]
This error had been reported for [arch](https://www.archlinux.org/) linux. The build fails as follows:
```
make -j8
[  2%] Built target gflags-static
[ 26%] Built target libprotobuf
[ 48%] Built target libprotoc
[ 76%] Built target artm-static
[ 82%] Built target google-glog
[ 84%] Built target protoc
[ 85%] Linking CXX executable bigartm
[ 86%] Built target artm
[ 98%] Built target artm_tests
/usr/bin/ld: cannot find -lunwind
collect2: error: ld returned 1 exit status
src/bigartm/CMakeFiles/bigartm.dir/build.make:158:
recipe for target 'src/bigartm/bigartm' failed
make[2]: *** [src/bigartm/bigartm] Error 1
CMakeFiles/Makefile2:624: recipe for target 'src/bigartm/CMakeFiles/bigartm.dir/all' failed
make[1]: *** [src/bigartm/CMakeFiles/bigartm.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2
```
This issue seems to be related to static linkage of bigartm CLI, introduced by https://github.com/bigartm/bigartm/commit/bae216ac7254ea271f01a7df1f9009f94dae6c43.

@JeanPaulShapo Could you please add CMake [option](https://cmake.org/cmake/help/v3.0/command/option.html) that enables static linkage for bigartm executable? This option needs to be ON by default. User that has the issue reported above can switch it OFF as a workaround. This is just a workaround for now, and eventually we may find someone with *arch* linux who could help us investigate why `bigartm` binary depends on `unwind` library (which we don't expect).

**Comment by JeanPaulShapo** [comment]
I managed to reproduce the bug on my laptop with Fedora. It depends on wheter you have libunwind.h files installed (headers for unwind library).

The dependency comes from GLOG library: they have configuration files to choose which libraries they can use to manipulate with stack.

So, -lunwind is dependency of GLOG library

**Comment by JeanPaulShapo** [comment]
Fixed by #359
